packages/core/chrome/core-chrome-browser/src/contracts.ts

```diff
@@ -137,4 +137,12 @@ export interface ChromeStart {
    * Get an observable of the current header banner presence state.
    */
   hasHeaderBanner$(): Observable<boolean>;
+
+  setCustomLogo(customLogo: string): void;
+
+  registerWhitelabellingPlugin(pluginName: string): void;
+
+  getCustomLogo$(): Observable<string | undefined>;
+
+  setCustomMark(custoMark: string): void;
```

Comment:
Two remarks here:

#### 1. I think we may want one single registration API to register all the white-labelling related information in a single call.

E.g instead of 

```ts
chrome.setCustomLogo(theming.logo);
chrome.setCustomMark(theming.mark);
```

I would rather have

```ts
chrome.setCustomThemeInfo(theming);
```

#### 2. I don't think `chrome` is the correct service for this/these registration APIs

I would expect these calls to be performed against the `theme` service (and then propagated from the theme service down to the services needing the info)

 ```ts
chrome.setCustomThemeInfo(theming);
```

=> 

```ts
theme.setCustomThemeInfo(theming);
```